components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ChefServerDetailsComponent } from './chef-server-details.component';
+import { HttpErrorResponse } from '@angular/common/http';
+import { RouterTestingModule } from '@angular/router/testing';
+import { ReactiveFormsModule, FormsModule } from '@angular/forms';
+import { MockComponent } from 'ng2-mock-component';
+import { StoreModule, Store } from '@ngrx/store';
+
+import { Org } from 'app/entities/orgs/org.model';
+import { CreateOrgFailure } from 'app/entities/orgs/org.actions';
+import { NgrxStateAtom, ngrxReducers, runtimeChecks } from 'app/ngrx.reducers';
+import { HttpStatus } from 'app/types/types';
+import { FeatureFlagsService } from 'app/services/feature-flags/feature-flags.service';
```

Comment:
Minor Chef convention is that imports should be organized from most general to most specific, the first section starting with all `@angular` items, then `@ngrx` items, then other 3rd party items (e.g. ng2-mock-component).
The second section, Chef files, should be similarly arranged, so `app/types/types` and `app/ngrx.reducers` at the top, the others follow.